docker/docker.go

```diff
@@ -69,12 +69,12 @@ func main() {
 	if err := c.Run(flag.Args()...); err != nil {
 		if sterr, ok := err.(cli.StatusError); ok {
 			if sterr.Status != "" {
-				fmt.Fprintln(os.Stderr, sterr.Status)
+				fmt.Fprintln(stderr, sterr.Status)
```

Comment:
it doesn't really matter either way. `stderr` will use an ansi emulator on windows clients, even though here we are not printing anything fancy.